hwdb.d/70-mouse.hwdb

```diff
@@ -143,6 +143,34 @@ mouse:*:name:*trackball*:*
 mouse:*:name:*TrackBall*:*
  ID_INPUT_TRACKBALL=1
 
+##########################################
+# 3D Connexion
+##########################################
+# From https://github.com/FreeSpacenav/spacenavd/blob/39856625a6de1e8c4b57c5938e1bf29d13cf1a9f/src/dev.c#L63
+mouse:usb:v046dpc603:name:*:*
+mouse:usb:v046dpc605:name:*:*
+mouse:usb:v046dpc606:name:*:*
+mouse:usb:v046dpc621:name:*:*
+mouse:usb:v046dpc623:name:*:*
+mouse:usb:v046dpc625:name:*:*
+mouse:usb:v046dpc626:name:*:*
+mouse:usb:v046dpc627:name:*:*
+mouse:usb:v046dpc628:name:*:*
+mouse:usb:v046dpc629:name:*:*
+mouse:usb:v046dpc62b:name:*:*
+mouse:usb:v046dpc640:name:*:*
+mouse:usb:v256fpc62e:name:*:*
+mouse:usb:v256fpc62f:name:*:*
+mouse:usb:v256fpc631:name:*:*
+mouse:usb:v256fpc632:name:*:*
+mouse:usb:v256fpc633:name:*:*
+mouse:usb:v256fpc635:name:*:*
+mouse:usb:v256fpc636:name:*:*
+ ID_3D_MOUSE=1
```

Comment:
Took me a while to figure out why it looks wrong: this should be `ID_INPUT_3D_MOUSE` to match the current convention.

Reply:
Doh! Fixed